src/components/Button/Button.tsx

```diff
@@ -37,25 +38,27 @@ const Button = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
     labelSuffixA11y,
     shouldHideLabel,
     icon,
+    subjectIcon,
     "aria-label": ariaLabel,
     htmlButtonProps = {},
     iconBackground,
     $font,
     disabled,
+    title,
     ...spacingProps
   } = props;
 
   const { size, variant, $iconPosition, background } =
     getButtonStylesProps(props);
 
   const defaultTitle =
-    ariaLabel || labelSuffixA11y ? `${label} ${labelSuffixA11y}` : "";
+    ariaLabel ?? (labelSuffixA11y && `${label} ${labelSuffixA11y}`) ?? label;
 
   return (
     <StyledButton
       ref={ref}
       {...htmlButtonProps}
-      title={htmlButtonProps.title || defaultTitle}
+      title={title ?? htmlButtonProps.title ?? defaultTitle}
```

Comment:
I think this could just be the previous version? `||` would make more sense that `??` as title is null before definition anyway I think

Reply:
Actually title can be defined on the button as an optional param